app/routes/routes.py

```diff
@@ -0,0 +1,119 @@
+from flask import Blueprint,jsonify, abort, make_response,request
+from app import db
+from app.models.task import Task
+from app.models.goal import Goal
+from dotenv import load_dotenv
+
+
+
+goals_bp = Blueprint ("goals", __name__, url_prefix="/goals")
+
+
+def handle_id_request(cls,id):
+
+    try:
+        id = int(id)
+    except:
+        abort(make_response(jsonify({"message":f"{cls.__name__} {id} invalid"}),400))
+    Object = cls.query.get(id)
```

Comment:
Here, you're reassigning the value of `Object` to `cls.query.get(id)`. I think it's really important here to _not_ re-assign `Object`, but to create a new variable. `model` might be a good name, since it's to represent either the `Task` model or the `Goal` model.